backend/src/main/java/com/makro/model/SimpleTime.java

```diff
@@ -0,0 +1,29 @@
+package com.makro.model;
+
+public class SimpleTime {
+    private String username;
+    private float time;
+
+    public SimpleTime(String username, float time) {
+        this.username = username;
+        this.time = time;
+    }
+
+    public SimpleTime withUsername(String username) {
+        this.username = username;
+        return this;
+    }
+
+    public SimpleTime withTime(float time) {
+        this.time = time;
+        return this;
+    }
```

Comment:
po co się to robi?

Reply:
Right now we can write: new SimpleTime().withUsername("Julia").withTime(2.15);